packages/@aws-cdk/aws-apigateway/lib/cors.ts

```diff
@@ -0,0 +1,105 @@
+import { Duration } from '@aws-cdk/core';
+import { ALL_METHODS } from './util';
+
+export interface CorsOptions {
+  /**
+   * Specifies the response status code returned from the OPTIONS method.
+   *
+   * @default 204
+   */
+  readonly statusCode?: number;
+
+  /**
+   * The Access-Control-Allow-Origin response header indicates whether the
+   * response can be shared with requesting code from the given origin.
+   *
+   * Specifies the list of origins that are allowed to make requests to this resource.
+   *
+   * @see https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Access-Control-Allow-Origin
```

Comment:
That documentation seems to indicate this can only be `*`, a single origin, or nothing. But here, you're accepting an array, and at this stage it's not clear what this implies.

Reply:
Fixed